runtime/autoload/ccomplete.lua

```diff
@@ -26,7 +27,7 @@ local SearchMembers = nil
 -- # Vim completion script
 -- # Language:	C
 -- # Maintainer:	The Vim Project <https://github.com/vim/vim>
--- # Last Change:	2023 Aug 10
+-- # Last Change:	2024 Jun 06
```

Comment:
👀

Reply:
I think it just uses the the same date that's in the vim file used to generate it: https://github.com/dundargoc/vim/blob/76bdb82527a13b5b2baa8f7d7ce14b4d5dc05b82/runtime/autoload/ccomplete.vim#L6

Reply:
Yes, but why was this changed _in this PR_?